plugins/woocommerce-blocks/tests/e2e/plugins/order-confirmation-filters.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Plugin Name: WooCommerce Blocks Order Confirmation Filters
+ * Description: used to modify filters and actions present in the new Order Confirmation Template
+ * Plugin URI: https://github.com/woocommerce/woocommerce
+ * Author: WooCommerce
+ *
+ * @package woocommerce-order-confirmation-filters
```

Comment:
For consistency's sake, let's also make sure this matches:

```suggestion
 * @package woocommerce-blocks-order-confirmation-filters
```